generator/transit_generator.cpp

```diff
@@ -4,43 +4,34 @@
 #include "generator/borders_loader.hpp"
 #include "generator/utils.hpp"
 
-#include "traffic/traffic_cache.hpp"
-
 #include "routing/index_router.hpp"
 #include "routing/routing_exceptions.hpp"
+#include "routing/routing_exceptions.hpp"
```

Comment:
Am I right that this line is duplicated?

Reply:
Yes. Thanks.